_includes/sidebar.html

```diff
@@ -77,7 +77,7 @@
                                     {% for deeplevel in thirdlevel.thirdlevelitems %}
                                         {% if deeplevel.external_url %}
                                         <li><a href="{{deeplevel.external_url}}" target="_blank">{{deeplevel.title}}</a></li>
-                                        {% elsif page.url == deeplevel.url %}
+                                        {% elsif page.url == deeplevel.url or deeplevel.alternate_urls contains page.url %}
```

Comment:
I would recommend splitting the pages up now. I'm not actively working on hugo, and it will be easier during the hugo work if they are already separate pages.